donations/management/commands/donations_mails.py

```diff
@@ -39,59 +39,90 @@ def handle(self, **options):
-
-        # Send email to users that downloaded more than X sounds in the last Y days
-        # excluding uploaders and excluding users that donated in less than Z days.
-        email_time_span = datetime.datetime.now()-datetime.timedelta(days=donation_settings.minimum_days_since_last_donation_email)
-        email_sent = User.objects.filter(profile__last_donation_email_sent__gt=email_time_span)
-        donations = Donation.objects.filter(user__isnull=False, created__gt=donation_time_span).values_list('user_id', flat=True)
-        # Get number of downloads in last Z days for each user excluding those that donated and those that received an email
-        users_downloads = Download.objects.filter(created__gte=email_time_span).exclude(user__in=email_sent)\
-                .exclude(user_id__in=donations).values('user_id').annotate(num_download=Count('user_id'))\
-                .order_by('num_download')
-        for user in users_downloads.all():
-            if user['num_download'] > donation_settings.downloads_in_period:
-                user = User.objects.get(id=user['user_id'])
-                # Check if the user is an uploader
-                if not donation_settings.never_send_email_to_uploaders or user.profile.num_sounds == 0:
-                    # If the user has donated in the past count the number of downloads after the donation "expired"
-                    last_donation = Donation.objects.filter(user=user, created__gt=donation_time_span).order_by('-created')
-                    user_downloads = None
-                    if last_donation.count() > 0:
-                        donation = last_donation.first()
-                        if donation.created > email_time_span:
-                            downloads_date_interval = donation.created
-                        else:
-                            downloads_date_interval = email_time_span
-                        user_downloads = Download.objects.filter(created__gte=downloads_date_interval, user=user).count()
-                    if last_donation.count() == 0 or user_downloads > donation_settings.downloads_in_period:
-                        send_mail_template(
-                            u'Donation',
-                            'donations/email_donation_request.txt', {
-                                'user': user,
-                                }, None, user.email)
-                        user.profile.last_donation_email_sent = datetime.datetime.now()
-                        user.profile.save()
-                        logger.info("Donation request sent to user %i" % user.id)
+                        'donations/email_donation_request.txt', {
+                            'user': user,
+                            }, None, user.email)
+                    user.profile.last_donation_email_sent = datetime.datetime.now()
+                    user.profile.save()
+                    logger.info("Donation request sent to user %i" % user.id)
```

Comment:
The text of the log message should be similar to the aboce.
Could be "Sent donations email (max downloads) to user:#"
Then the one above could be changed to: "Sent donations email(reminder) to...".
In this way the reason is included in the message.